src/components/AddSensorPage.vue

```diff
@@ -1,11 +1,14 @@
 <template>
-  <v-container>
-    <v-card elevation="5">
-      <v-row>
-        <v-col>
-          <p class="text-h2 text-center">Add a Sensor</p>
-        </v-col>
-      </v-row>
+  <v-container class="d-flex justify-center">
+    <v-card
+      elevation="5"
+      rounded="lg"
+      color="green lighten-2"
+      max-width="700"
+      class="py-4"
+    >
+      <p class="px-2 text-h2 text-center">Add a Sensor</p>
+
```

Comment:
I would remove the d-flex and max-width styling, because that changes the width of the card when you choose a different gps option.